### ISSUE TITLE
replay causing parser crash

### ISSUE THREAD

**Issue by howardchung** [odota/core#1526]
For some reason this replay crashes odota/parser.

Seems to be some value getting set to Infinity for some reason.

https://github.com/odota/testfiles/blob/master/3397774315_38200034.dem.bz2

**Comment by PavelCore** [comment]
Yeah, i used GsonBuilder with serializeSpecialFloatingPointValues to catch this and found that field teamfight_participation got infinity. I think, appropriate solution is to replace infinities in getEntityProperty in Parse.java
I can make pr today, i guess
```
"teamfight_participation":Infinity
```
